tree/20_cfe_basics/dry_run_lib.cf

```diff
@@ -82,6 +82,71 @@ bundle agent set_dry_run_mode(mode)
       "${configuration.debug} Enabling dry-run mode";
 }
 
+bundle agent dry_run_unique
+{
+  vars:
+      "value" string => "1";
+      "stack" slist => {};
+}
+
```

Comment:
Could you add a usage doc in a comment: how to initialize the stack and use it afterwards, what happens if you use set_dry_run_mode with oush/pop, etc.